schema.graphql

```diff
@@ -213,3 +217,8 @@ type Event @entity {
   "Account of the payer - Related to Repay event"
   payer: Account
 }
+
+type Contract @entity {
+  id: ID!
+  name: String
+}
```

Comment:
did you intend to keep this in the schema? Or were you testing it to see the Pool Configurator and Pool Proxy in the query results?